syft/execution/role.py

```diff
@@ -122,6 +122,20 @@ def _instantiate_inputs(self, args):
         )
         PlaceHolder.instantiate_placeholders(input_placeholders, args)
 
+        # Last extra argument is a state?
```

Comment:
Was that a pre-existing thing or something new from this PR?

Reply:
This is new. This allows PlanTranslatorTorchscript to pass Plan's state tensors as input arguments to a Plan, so they don't have to be recorded in torchscript as nn.Parameters.
Not elegant, but this should be refactored away when ComputationPhase is introduced.